src/map/npc.c

```diff
@@ -3724,7 +3724,7 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		return strchr(start,'\n');// skip and continue
```

Comment:
A few lines above the format is in the comments. It also needs to be adjusted to the new one.
Can't comment on the correct line... >_>